components/atoms/accordion-section/index.js

```diff
@@ -0,0 +1,64 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import classnames from 'classnames'
+
+import styles from './styles.css'
+
+class Section extends Component {
+  state = {
+    isOpen: true,
+    bodyHeight: 'auto',
+  }
+
+  handleClick = () => {
+    const { isOpen } = this.state
+    this.setState({
+      isOpen: !isOpen,
+    })
+  }
+
+  bodyRef = element => {
+    const bodyRect = element.getBoundingClientRect()
+    this.setState({ bodyHeight: bodyRect.height, isOpen: false })
+  }
+
+  render = () => {
+    const { isOpen, bodyHeight } = this.state
+    const height = isOpen ? bodyHeight : 0
+    const iconClass = classnames(styles.downIcon, {
+      [styles.exitIcon]: isOpen,
+    })
+    const title = classnames(styles.title, {
+      [styles.selected]: isOpen,
+    })
+
+    return (
+      <div className={styles.section}>
+        <button className={styles.header} onClick={this.handleClick}>
+          <h1 className={title}>{this.props.sectionTitle}</h1>
+          <i className={iconClass} />
+        </button>
+        <div className={styles.body} ref={this.bodyRef} style={{ height }}>
```

Comment:
I don't know if it's a good idea to have className and style both as props
We can talk more about this with Doug, DLeite and Mibs